resources/sdk-modules.md

```diff
@@ -0,0 +1,169 @@
+Modules to use with Swedbank Pay APIs." %}
+
+## Introduction
+
+Swedbank Pay offers APIs, SDKs, libraries, modules, extensions and plugins as
+different ways to integrate and use our products and services. It might not
+be obvious to everyone what these are, so to describe them we use the analogy
+of **baking a cake**.
+
+### APIs
+
+APIs (Application Programming Interface) are the lowest level building block of
+our offering and can be compared to **raw ingredients**. They give you a whole
+lot of flexibility, but also requires you to have more knowledge in how to use
+them correctly and you need to know which recipe to cook in order to get a
+workable cake out in the other end. To efficiently use an API, you need to be
+a trained baker.
+
+### SDKs
+
+SDKs (Software Development Kit), also known as libraries, can be compared to
+a **cake mix**. It requires less knowledge of the ingredients and how to use
+them and a recipe is provided for you. Because of this, you also have less
+flexibility because you can't bake any cake, you can only bake the one the mix
+is for. You still have to mix everything together, so you need some experience
+in the kitchen is required, but less than using the "raw" API.
+
+### Modules
+
+Modules, also known as extensions or plugins, can be compared to a **readily
+baked cake** purchased at a bakery. You don't need any baking skills to have
+one, but you also have very little flexibility in what goes into the cake or
+how it tastes.
+
+## Official Modules
+
+**Modules, extensions and plugins** are typically used as a way to extend
+webshop platforms such as [WooCommerce][woocommerce] and [Magento][magento] with
+functionality that isn't provided in the core platform. Such functionality may
+be to allow consumers of your website to pay with a payment provider such as
+Swedbank Pay.
+
+{% include alert.html type="warning" icon="warning" header="Unsupported"
+body="These modules are at an early stage of development and are not supported
+as of yet by Swedbank Pay. They are provided as a convenience to speed up your
+development, so please feel free to play around. However, if you need support,
+please wait for a future, stable release." %}
+
+{:.table .table-striped}
+|       | Platform | Module | Repository |
+| :---: | :------- | :----- | :--------- |
+{%- for repository in active_repositories -%}
+  {%- if repository.topics contains 'module' %}
+|   {%- if repository.topics contains 'swedbank-pay-checkout' -%} {% icon shopping_cart %}
+    {%- elsif repository.topics contains 'swedbank-pay-payments' -%} {% icon credit_card %}
+    {%- else -%} ?
+    {%- endif -%}
+|   {% if repository.topics contains 'episerver' -%} Episerver
+    {%- elsif repository.topics contains 'magento2' -%} Magento 2
+    {%- elsif repository.topics contains 'woocommerce' -%} WooCommerce
+    {%- else -%} ?
+    {%- endif -%}
+|   {% if repository.homepage != empty -%}[{{ repository.description }}]({{ repository.homepage }})
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)